Maui.RevenueCat.InAppBilling/Platforms/Android/RevenueCatBillingAndroid.cs

```diff
@@ -56,48 +56,47 @@ public partial class RevenueCatBilling : IRevenueCatBilling
 
     public async partial Task<List<OfferingDto>> LoadOfferings(bool forceRefresh, CancellationToken cancellationToken)
     {
-        if (!forceRefresh && !_cachedOfferingPackages.IsNullOrEmpty())
+        if (!forceRefresh && _cachedOfferingPackages != null)
         {
-            return _cachedOfferingPackages.ToOfferDtoList(); ;
+            return _cachedOfferingPackages.ToOfferingDtoList().ToList();
```

Comment:
is the second `ToList` really necessary ?

Reply:
Yes because it returns List instead of IList, this is related to the previous IList discussion.  I didn't fix it because I didn't want to change the established interface.